seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json

```diff
@@ -36,7 +36,7 @@
       "id": "bd7158d8c443edefaeb5bdef",
       "title": "Timestamp Microservice",
       "description": [
-        "Build a full stack JavaScript app that is functionally similar to this: <a href='https://curse-arrow.gomix.me/' target='_blank'>https://curse-arrow.gomix.me/</a> on GoMix or on your own publicly accessible domain.",
+        "Build a full stack JavaScript app that is functionally similar to this: <a href='https://curse-arrow.gomix.com/' target='_blank'>https://curse-arrow.gomix.com/</a> on GoMix or on your own publicly accessible domain.",
```

Comment:
https://curse-arrow.gomix.com doesn't work for me. https://curse-arrow.gomix.me does.